lib/cli/writer.js

```diff
@@ -23,7 +23,7 @@ const os = require('os');
 const unmount = require('./unmount');
 const imageStream = require('../image-stream');
 const errors = require('../shared/errors');
-const constraints = require('../shared/constraints');
+const constraints = require('../shared/drive-constraints');
```

Comment:
I wish we could re-use auto hat. I spoke with the team at some point, but they would only support GNU/Linux :(